modules/bootstrapped/src/generated/smithy4s/example/product/ExampleOperationInput.scala

```diff
@@ -0,0 +1,23 @@
+package smithy4s.example.product
```

Comment:
**thought:** side note, I think the recent codegen changes around the `bootstrapped` module could've broken something wrt. detecting stale files - e.g. if you delete a shape from the smithy specs, you need to run a `clean` in order for that to be reflected on the filesystem

Reply:
>  e.g. if you delete a shape from the smithy specs, you need to run a clean in order for that to be reflected on the filesystem 

That is correct. But at least now it's wired in `clean`, previously you had to manually remove stuff.